### PR TITLE
Main view scripts fixes, popup fixes, styling fixes

### PR DESCRIPTION
**Pull request by Flight**
Main view scripts fixes, popup fixes, styling fixes.

**Please do not review widget.js**, it's just restored.

### CODE REVIEW

**Comment by FinalAngel** [comment]
LGTM

**Comment by Brankub** [comment]
:+1:

File: filer/static/filer/js/addons/widget.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        var clearer = $(this);
+        var hidden_input = clearer.closest('.filerFile').find('input.vForeignKeyRawIdAdminField');
+        var base_id = '#' + hidden_input.attr('id');
+        var thumbnail = $(base_id + '_thumbnail_img');
+        var description = $(base_id + '_description_txt');
+        var static_prefix = clearer.attr('src').replace('admin/img/icon_deletelink.gif', 'filer/');
+
+        clearer.hide();
+        hidden_input.removeAttr('value');
+        thumbnail.attr('src', static_prefix + 'icons/nofile_48x48.png');
```
Comment (dhorelik) [inline filer/static/filer/js/addons/widget.js]: `static_prefix` is going to end with `filer/` anyway, can we just use it here and just clean the `clearer.attr('src')` above? Maybe I'm missing something, though

**Comment by vxsx** [comment]
:+1:

File: filer/static/filer/js/addons/widget.js (hunk 1 above)
Comment (dhorelik) [inline filer/static/filer/js/addons/widget.js]: ouch, missed the description..

**Pull request merged**